test/phoenix_live_view/diff_test.exs

```diff
@@ -293,6 +293,8 @@ defmodule Phoenix.LiveView.DiffTest do
     end
   end
 
+  # TODO
```

Comment:
TODO What? :)

Reply:
To pay attention when reviewing the code before submitting the PR so you don't push useless TODOs :)